nodejs.org/locale/uk/security.md

```diff
@@ -1,61 +1,48 @@
 ---
```

Comment:
@nodejs/nodejs-uk please, anyone check this file. I'm not sure this is a correct translation, especially 
juridical terms.